stable/kiam/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: kiam
-version: 1.2.0
-appVersion: 2.8
+version: 2.0.0
```

Comment:
Does the app upgrade make this a breaking change in the semver sense? If so this is 👍 If not this should bump MINOR version only.

Reply:
@scottrigby Yes, it was my feeling that this isn't a semver breaking change for the chart - I think any custom `values.yaml` that worked with chart v1.2.0 will continue to work with this newer chart version.